topology.yaml

```diff
@@ -0,0 +1,134 @@
+entrypoints:
+- identifier: 'Microsoft.Azure.ARO.HCP.Global'
+  metadata:
+    name: Global
+    scopeDoc: high-level-architecture.md
+- identifier: 'Microsoft.Azure.ARO.HCP.Region'
+  metadata:
+    name: Region
+    scopeDoc: high-level-architecture.md#regional-scope
+- identifier: 'Microsoft.Azure.ARO.HCP.Service.Infra'
+  metadata:
+    name: Service Cluster
+    scopeDoc: high-level-architecture.md#service-cluster
+- identifier: 'Microsoft.Azure.ARO.HCP.Management.Infra'
+  metadata:
+    name: Management Cluster
+    scopeDoc: high-level-architecture.md#management-clusters
+services:
+- serviceGroup: Microsoft.Azure.ARO.HCP.Global
+  children:
+  - serviceGroup: Microsoft.Azure.ARO.HCP.Region
+    children:
+    - serviceGroup: Microsoft.Azure.ARO.HCP.Service.Infra
+      children:
+      - serviceGroup: Microsoft.Azure.ARO.HCP.ACRPull
+        metadata:
+          pipeline: acrpull/pipeline.yaml
+          purpose: 'Manage ACR pull credentials.'
+          intPipelineLink: ''
+          stgPipelineLink: ''
+          prodPipelineLink: ''
+      - serviceGroup: Microsoft.Azure.ARO.HCP.Maestro.Server
+        metadata:
+          pipeline: maestro/server/pipeline.yaml
+          purpose: 'Deploy the Maestro Server.'
+          intPipelineLink: 'https://msazure.visualstudio.com/AzureRedHatOpenShift/_build?definitionId=382258'
```

Comment:
instead of copy pasting the entire link, perhaps add the definition ID only and have the rest a variable and build the link in the parser?

Reply:
Smart - will do